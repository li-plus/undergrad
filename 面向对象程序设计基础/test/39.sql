create database oop;
use oop;
create table t(a text); 
insert into t(a) values("hello world"); 
insert into t(a) values("goodbye world");
insert into t(a) values("hell no");
insert into t(a) values("good morning");
insert into t(a) values("hello+-*()[]{}^.?$|\/"); 
select a from t where a like "hell%"; 
/********************** Sample Output *************************
a       
hell no 
hello world     
hello+-*()[]{}^.?$|\/   
**************************************************************/
select a from t where a like "%wor%";
/********************** Sample Output *************************
a       
goodbye world   
hello world 
**************************************************************/
select a from t where a like "%+-*()[]{}^.?$|\/";
/********************** Sample Output *************************
a       
hello+-*()[]{}^.?$|\/   
**************************************************************/
drop database oop;