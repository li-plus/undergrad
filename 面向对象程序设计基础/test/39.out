a       
hell no 
hello world     
hello+-*()[]{}^.?$|\/   
a       
goodbye world   
hello world 
a       
hello+-*()[]{}^.?$|\/   
